internal/services/network/subnet_resource.go

```diff
@@ -59,6 +59,7 @@ var subnetDelegationServiceNames = []string{
 	"Microsoft.DBforPostgreSQL/singleServers",
 	"Microsoft.DelegatedNetwork/controller",
 	"Microsoft.DevCenter/networkConnection",
+	"Microsoft.DevOpsInfrastructure",
```

Comment:
```suggestion
	"Microsoft.DevOpsInfrastructure/pools",
```

same in another change as well

Reply:
thanks :)